style.css

```diff
@@ -0,0 +1,50 @@
+*{
+    margin:0;
+    padding:0;
+}
+
+header{
+    font-size: 25px;
+    font-weight:bold;
+}
+
+
+
+
+
+
+
+
```

Comment:
It seems you have empty spaces in your style file. It is best to use Linters to correct that, so you have a clean code. 😉